Tasks/NpmAuthenticateV0/task.json

```diff
@@ -74,6 +74,10 @@
         "RevertedChangesToNpmrc": "Reverted changes made to %s.",
         "NpmrcDoesNotExist": "The .npmrc file you selected at %s does not currently exist.",
         "SuccessfulPush": "Successfully pushed .npmrc",
-        "SuccessfulAppend": "Successfully appended .npmrc"
+        "SuccessfulAppend": "Successfully appended .npmrc",
```

Comment:
I believe these should also be added to the "NpmAuthenticateV0\Strings\resources.resjson\en-US\resources.resjson" file to be localized

Reply:
I may be wrong about this though.

Reply:
I think I fixed it